src/components/spaces/organisms/organelle-wrapper.tsx

```diff
@@ -100,10 +102,22 @@ export class OrganelleWrapper extends BaseComponent<OrganelleWrapperProps, Organ
   }
 
   public componentWillReact() {
-    const zoomLevel = this.props.zoomLevel;
-    if (this.model && zoomLevel !== this.model.zoomLevel) {
-      this.createNewModel();
+    const { zoomLevel, mode } = this.props;
+    if (this.model) {
+      if (zoomLevel !== this.model.zoomLevel) {
+        this.createNewModel();
+      }
+
+      if (mode !== this.model.mode) {
```

Comment:
I wasn't sure what the best way to pause the model was so I followed your lead here, let me know what you think.

Reply:
Yeah, I don't know whether there's a better way to do this. I basically wanted the equivalent of `componentDidUpdate(prevProps)` so that we could compare the props before and after, and call these functions on the model, but I couldn't see how MST supports that. So I stuck the `zoomLevel` prop on the model just as a way of caching the value. It could have also just been a `let` outside the class. It's not the right solution, I think, but I couldn't work out the right solution.